simulator/docs/env-variables.md

```diff
@@ -0,0 +1,36 @@
+# Different Environment Variables used
+
+This page describes about the different environment variables that are
+used to configure kube-scheduler-simulator. 
+
+Please refer [docker-compose.yml](./docker-compose.yml) as an example
+use.
+
+`Port`: (required) This is the port number on which kube-scheduler-simulator
+server is started.
+
+`KUBE_SCHEDULER_SIMULATOR_ETCD_URL`: (required) This is the URL for
+etcd. The simulator runs kube-apiserver internally, and the
+kube-apiserver uses this etcd.
+
+`FrontendURL`: This URL represents the URL web UI started. The
+simulator and internal kube-apiserver set the origin as the allowed
+origin for `CORS_ALLOWED_ORIGIN_LIST`.
+
+`KUBECONFIG`: This is for the beta feature "Existing cluster
+Importing". This variable is used to find Kubeconfig required to
+access your cluster for importing resources to scheduler simulator.
+
+`KUBE_API_HOST`: This is the host of kube-apiserver which the
+simulator starts internally.
+
+`KUBE_SCHEDULER_CONFIG_PATH`: The path to a KubeSchedulerConfiguration
+file.  If passed, the simulator will start the scheduler with that
+configuration.  Or, if you use web UI, you can change the
+configuration from the web UI as well.
+
+`ExternalImportEnabled`: This variable indicates whether the simulator will import resources from an existing cluster or not.
+
+## For Web UI
+
```

Comment:
Also, for web ui, ALPHA_TABLE_VIEWS and BASE_URL should be mentioned.
They do the followings. I am not a good English speaker, feel free to change the wording as you like.

BASE_URL → the URL for the simulated server.
ALPHA_TABLE_VIEWS → The value is 0 or 1. (0 means disabled, 1 means enabled.) Enable the alpha feature "table view". You can see the resource status in the table.

Reply:
Updated as suggested above.